### PR TITLE
HARMONY-1439: Removed collection references (except for granule limit) used by harmony tests from services.yml

### PR DESCRIPTION
**Pull request by ygliuvt**
## Jira Issue ID
HARMONY-1439

## Description
Removed collection references (except for granule limit) used by harmony tests from services.yml.
Fixed a couple places where we don't handle collections that have Harmony services (via umm-s associations in services.yml) correctly.

## Local Test Steps
Verify there is no collections referenced in Harmony tests that are in the collections list in service.yml and all tests are passing.

## PR Acceptance Checklist
* [x] Acceptance criteria met
* [x] Tests added/updated (if needed) and passing
* [ ] Documentation updated (if needed)

### CODE REVIEW

File: env-defaults [hunk 1]
```diff
@@ -143,10 +143,10 @@ DATABASE_URL=
 UPLOAD_BUCKET=local-upload-bucket
 
 # SQS queue used to process work item updates
-WORK_ITEM_UPDATE_QUEUE_URL=http://localstack:4566/queue/work-item-update-queue
+WORK_ITEM_UPDATE_QUEUE_URL=http://localhost:4566/queue/work-item-update-queue
```
Comment (chris-durbin) [inline env-defaults:146]: I think this accidentally reverted a change James made to fix harmony in a box.

File: test/ogc-api-coverages/get-coverage-rangeset.ts [hunk 2]
```diff
@@ -1164,7 +1164,7 @@ describe('OGC API Coverages - getCoverageRangeset with a collection not configur
   });
 
   describe('when using short name for a collection with granules but no services configured', function () {
-    const shortName = 'GLOBAL_MICROBIAL_BIOMASS_C_N_P_1264';
+    const shortName = 'harmony-SEA_SURFACE_HEIGHT_ALT_GRIDS_L4_2SATS_5DAY_6THDEG_V_JPL1812';
```
Comment (chris-durbin) [inline test/ogc-api-coverages/get-coverage-rangeset.ts:1167]: awesome - thanks for fixing

File: app/models/services/index.ts [hunk 3]
```diff
@@ -104,6 +104,20 @@ const serviceTypesToServiceClasses = {
...
+ * For a given list of collections, return a list of collection concept ids that have Harmony serices defined via
+ * the umm-s associations in services.yml. This is used to filter out any collections that do not have services associated,
+ * so we only deal with collections that are applicable to Harmony services.
+ * @param collections - an initial list of collections
+ * @returns a list of collection concept ids that have Harmony serices
+ */
+export function harmonyCollections(
+  collections: CmrCollection[],
+): string[] {
+  const allServiceConfigs = addCollectionsToServicesByAssociation(collections);
```
Comment (chris-durbin) [inline app/models/services/index.ts:117]: Nice clean fix - I see you didn't even have to change the WMS code as a result.

**Review by chris-durbin — APPROVED** [review]
Make sure to revert the change to `env-defaults` before merging.

File: env-defaults (hunk 1 above)
Comment (indiejames) [reply env-defaults:146]: Yes, it did.

File: env-defaults [hunk 4]
```diff
@@ -143,10 +143,10 @@ DATABASE_URL=
...
 
 # SQS queue used to process work item updates
-LARGE_WORK_ITEM_UPDATE_QUEUE_URL=http://localstack:4566/queue/large-work-item-update-queue
+LARGE_WORK_ITEM_UPDATE_QUEUE_URL=http://localhost:4566/queue/large-work-item-update-queue
```
Comment (indiejames) [inline env-defaults:149]: Please put these back so as to not break harmony-in-a-box. You can add them to your .env file.

**Review by indiejames — CHANGES_REQUESTED** [review]
Looks good except for the regression on the queue urls in env-defaults

File: env-defaults (hunk 4 above)
Comment (ygliuvt) [reply env-defaults:149]: Thanks! Didn't realize it was for harmony in a box.

**Review by indiejames — APPROVED** [review]

**Pull request merged**